source/Magritte-Pharo7-Model/MAFileDescription.extension.st

```diff
@@ -0,0 +1,14 @@
+Extension { #name : #MAFileDescription }
```

Comment:
I'm not especially familiar with this corner of the system, but I think the problem is the baseline, not the packaging. IFAICT, `Magritte-Pharo7-Model` is a platform compatibility layer (and there would be one for Squeak, GemStone, etc). As such, I think it needs to be loaded *before* `Magritte-Model` and I think that solves the problem, right? I don't see any Magritte-specific stuff in the Pharo7 package.

So:
```smalltalk
spec
				package: 'Magritte-Pharo7-Model' with: [ spec requires: #('Magritte-Model') ];
				package: 'Magritte-Tests-Pharo-Model' with: [ spec requires: #('Magritte-Pharo7-Model') ];
				package: 'Magritte-Model' with: [ spec includes: #('Magritte-Pharo7-Model') ];
				" create a temporary alias "
				package: 'Magritte-Pharo-Model' with: 'Magritte-Pharo7-Model'
```
should become
```smalltalk
spec
				package: 'Magritte-Pharo7-Model'; "remove requires"
				package: 'Magritte-Tests-Pharo-Model' with: [ spec requires: #('Magritte-Pharo7-Model') ];
				package: 'Magritte-Model' with: [ spec requires: #('Magritte-Pharo7-Model') ]; "change includes to requires"
				" create a temporary alias "
				package: 'Magritte-Pharo-Model' with: 'Magritte-Pharo7-Model'
```

Reply:
Flipping the dependency didn't work because MAExternalFileModel depends on MAFileModel (superclass). Although I can't see why the dependency is circular. A requires B and B includes A, which is fine. What am I missing?